app/models/employment/DecodedCreateNewEmploymentDetailsPayload.scala

```diff
@@ -31,4 +31,4 @@ case class DecodedCreateNewEmploymentDetailsPayload(employmentData: DecodedNewEm
 
 object DecodedCreateNewEmploymentDetailsPayload {
   implicit def writes: OWrites[DecodedCreateNewEmploymentDetailsPayload] = Json.writes[DecodedCreateNewEmploymentDetailsPayload]
-}
\ No newline at end of file
+}
```

Comment:
empty line

Reply:
Yes that was deliberate. I added a few like that to remove the Scala style warnings that specify the rule the end of file should have an empty line.

I can’t see this. Maybe it was already removed by my rebase as there is only one?